addons/website_slides/models/slide_channel.py

```diff
@@ -153,7 +156,7 @@ def _default_access_token(self):
     is_member = fields.Boolean(string='Is Member', compute='_compute_is_member')
     channel_partner_ids = fields.One2many('slide.channel.partner', 'channel_id', string='Members Information', groups='website.group_website_publisher')
     upload_group_ids = fields.Many2many(
-        'res.groups', 'rel_upload_groups', 'channel_id', 'group_id', string='Upload Groups',
+        'res.groups', 'rel_upload_groups', 'channel_id', 'group_id', string='New Content',
```

Comment:
That string does not mean anything ... I would keep the old one. If someone want to change it in the view, just update label, but model fields should be understandable ^^ .